app/models/listing.rb

```diff
@@ -132,17 +130,6 @@ def set_updates_email_at_to_now
   validates_inclusion_of :valid_until, :allow_nil => :true, :in => DateTime.now..DateTime.now + 7.months
   validates_numericality_of :price_cents, :only_integer => true, :greater_than_or_equal_to => 0, :message => "price must be numeric", :allow_nil => true
 
-  def set_community_visibilities
-    if current_community_id
```

Comment:
Everything should work as before. In practise this piece of code was use to add the `@current_community` to the `communities` relation. Now we achieve the same by setting the id of the `@current_community` to `listing.community_id`.